run_tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Test runner script for plenary tests
+set -e
+
+echo "Setting up plenary test environment..."
+
+# Download plenary if not present
+if [ ! -d "tests/plenary.nvim" ]; then
```

Comment:
Move plenary to vendor/plenary.nvim. this way we don't need to filter out plenary tests